### PR TITLE
Handle edge cases and add more verbose print statements for rack workflow

### PR DESCRIPTION
**Pull request by adhil0**
- Increase timeout from 5 to 20 secs
- Add more print statements to document what script is doing
- Handle edge cases where certain data can't be pulled from Sunbird

### CODE REVIEW

File: population/create_glpi_computer_redfish.py [hunk 1]
```diff
@@ -1043,7 +1045,11 @@ def add_rack_location_from_sunbird(
                 "No Sunbird config provided, setting data center and room automatically"
             )
             data_center = location_data["tiDataCenterName"]
-            room = location_data["tiRoomName"]
+            if "tiRoomName" in location_data:
+                room = location_data["tiRoomName"]
+            else:
+                # If the Room is not specified in Sunbird, use the Data Center name
+                room = location_data["tiDataCenterName"]
```
Comment (dkosteck) [inline population/create_glpi_computer_redfish.py:1052]: Any handling needed for a case where that's also not populated? Or is that a case that isn't really likely?

**Review by dkosteck — APPROVED** [review]
I think the verbosity is good, as is the addition of racks, just one comment to address but overall seems good

File: population/create_glpi_computer_redfish.py (hunk 1 above)
Comment (adhil0) [reply population/create_glpi_computer_redfish.py:1052]: Changed `data_center` to default to `None` if tiDataCenterName doesn't exist, and added an elif to check this.

Comment (dkosteck) [reply population/create_glpi_computer_redfish.py:1052]: @adhil0 Got it, makes sense

**Pull request merged**